src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsallocations/listener/OffenderEventListener.kt

```diff
@@ -10,18 +11,18 @@ import org.slf4j.LoggerFactory
 import org.springframework.stereotype.Component
 import uk.gov.justice.digital.hmpps.hmppsallocations.client.ForbiddenOffenderError
 import uk.gov.justice.digital.hmpps.hmppsallocations.service.UpsertUnallocatedCaseService
+import uk.gov.justice.hmpps.sqs.HmppsQueueService
 
 @Component
 class OffenderEventListener(
   private val objectMapper: ObjectMapper,
   private val upsertUnallocatedCaseService: UpsertUnallocatedCaseService,
+  private val hmppsQueueService: HmppsQueueService,
 ) {
 
   @SqsListener("hmppsoffenderqueue", factory = "hmppsQueueContainerFactoryProxy")
   fun processMessage(rawMessage: String) {
-    log.debug("processing message")
```

Comment:
Could we leave this log message in, or put it at beginning of getCrn function?
Just a suggestion.